onboarding/sprint-rotation.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Sprint Rotation
+description: What is the sprint rotation and how does it work?
+---
+
+## Sprint Rotation
+
+After an engineer has gone through basic onboarding (setting up systems, etc.) which will last about a week, they may spend a few sprints switching between different teams.
+
+### Why a sprint rotation?
+
+When engineers begin at Artsy, we want them to develop an understanding of our systems and business at a pace that fits their current experience. There's a lot of information to absorb and when engineers start on a product team with immediate sprint goals, there's often not the time or space to learn about the many facets of Artsy.
+
+The sprint rotation is an attempt to mitigate that worry, and also make onboarding/mentorship a shared responsibility. If an engineer switches teams at the start of a new quarter, they should be able to consume the same documentation or expertise that a team has developed for the sprint rotation.
+
+#### Goals
+- Get up-to-speed on Artsy's systems by being part of different product teams.
+- Learn about different sides of Artsy's business in a first-hand way.
+- Get to know our engineering team and how different engineers and teams work
+
+### How does the sprint rotation work?
+
+A new engineer's manager will work with tech leads to determine the proper length of the sprint rotation and which teams will be involved. This process can be used for the sole purpose of onboarding, and/or to help choose which team the new engineer will start on full-time.
+
+Following the sprint rotation, the new engineer will join a team full-time.
+
+#### Before the rotation starts
+- Manager schedules the duration and content of the sprint rotation, being cognizant of factors like:
+  - Other team members also going through sprint rotation
+  - Schedule of tech leads/PMs
+  - Specific areas of interest for new engineer
+
+#### Before the sprint starts
+_Manager responsibilities:_
+- Make sure the new engineer has met with the tech lead of the team for the upcoming sprint. This may have happened already during the first week.
```

Comment:
Does sprint rotation not start until after the onboarding week? If not, then does this mean that in the first pass they will be meeting the lead in the week that they’re also starting on that lead’s team?

Reply:
How about meeting with the lead _and_ PM? Combined they should be able to give the best representation of that team’s scope and goals.

Reply:
I like the suggestion to meet with the tech lead and PM (if possible).

Right now we try to schedule a bunch of meetings with tech leads in the first week before the sprint rotation starts, but @ansor4 brought up yesterday that the information is much more relevant when you get it just before the rotation.

Maybe instead of our current strategy we can do something like:
- First week: general meetings (intro to our engineering org/culture, intro to our product teams/structure at a high level, intro to FE practice/platform practice)
- Before a sprint rotation: meet with tech lead/PM

Reply:
I like that! 👌 It seems impossible to expect somebody to even get to do any real team product work in that first week.

Reply:
(Re that first week, more time can be spent on improving docs they’re reading https://github.com/artsy/README/pull/29)